src/test/resources/18_ignore_case/test_execution_response_FAIL.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
you can directly assign this to the string as this is very small length.
See- testIgnoreCaseWith_containsMatch()

Reply:
I have fixed it.